### PR TITLE
Use SQLite as DB

### PR DESCRIPTION
**Pull request by antoinechalifour**
- [x] Fix varieties (mega evos, ultras, ...) names and sprites
- [x] Debug perfs ?

-> About the perfs issues, it seems to come from the pokemon.moves resolver

### CODE REVIEW

File: dev-server/scripts/v2/migrations/20191226105119_import_types.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  ghost: "#66b",
+  normal: "#aa9"
+};
+
+const createTypeTable = knex =>
+  knex.schema.createTable("type", table => {
+    table.integer("id").primary();
+    table.string("color").notNullable();
+    table
+      .integer("name_en")
```
Comment (francoischalifour) [inline dev-server/scripts/v2/migrations/20191226105119_import_types.js]: Why is `name_en` an integer?

File: dev-server/scripts/v2/migrations/20191226115006_import_pokemon.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+const { getDirectoryContent } = require("../migrations_utils");
+
+const createPokemonTable = knex =>
+  knex.schema.createTable("pokemon", table => {
+    // --------------------------------------- //
+    // General info
+    // --------------------------------------- //
+    table
+      .integer("id")
```
Comment (francoischalifour) [inline dev-server/scripts/v2/migrations/20191226115006_import_pokemon.js:19]: Should this be a primary?

File: dev-server/scripts/v2/migrations/20191226170301_import_versions.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      .unsigned()
+      .notNullable()
+      .primary();
+    table.string("color").notNullable();
+    table.string("name_en").notNullable();
+  });
+
+const mapVersionToVersionDatabase = version => ({
+  id: version.id,
+  color: COLOR_BY_VERSION[version.name] || console.log(version) || null,
```
Comment (francoischalifour) [inline dev-server/scripts/v2/migrations/20191226170301_import_versions.js]:
```suggestion
  color: COLOR_BY_VERSION[version.name] || null,
```

File: dev-server/scripts/v2/migrations/20191226175528_import_abilities.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+const createAbilityTable = async knex =>
+  knex.schema.createTable("ability", table => {
+    table
+      .integer("id")
+      .unsigned()
+      .notNullable()
+      .primary();
+    table.string("name_en").notNullable();
+    table.string("name_fr");
```
Comment (francoischalifour) [inline dev-server/scripts/v2/migrations/20191226175528_import_abilities.js:17]: They should all have a French name.

```suggestion
    table.string("name_fr").notNullable();
```

File: dev-server/scripts/v2/migrations/20191226175528_import_abilities.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    table.string("name_ja");
```
Comment (francoischalifour) [inline dev-server/scripts/v2/migrations/20191226175528_import_abilities.js:18]: They all have a `ja` name but the language you're looking for is `roomaji`. Only the 7th generation doesn't have this property in our current dataset. We can scrap those later.

File: dev-server/src/graphql/resolvers/pokemon.js [hunk 6]
```diff
@@ -1,156 +1,51 @@
...
-  },
-  pokedexEntries: async pokemon => {
-    const species = await findPokemonSpeciesByPokemonId(pokemon.id);
+  moves: async ({ id }, args, { moveRepository }) => {
+    const moves = await moveRepository.findMovesByPokemonId(id);
 
-    return species.flavor_text_entries.filter(x => x.language.name === "en");
+    return moves.map(x => ({
+      ...x,
+      pokemon_id: id
```
Comment (francoischalifour) [inline dev-server/src/graphql/resolvers/pokemon.js:40]: This should probably be named `pokemonId` at this point.

**Review by francoischalifour — APPROVED** [review]
All the `ja` names should be replaced by `roomaji` at some point. (To make it simpler, we could call the table columns `{x}_ja` but use the `roomaji` value later.)

File: dev-server/scripts/v2/migrations/20191226175528_import_abilities.js (hunk 4 above)
Comment (antoinechalifour) [reply dev-server/scripts/v2/migrations/20191226175528_import_abilities.js:17]: I'm not sure we have all values for this :/

File: dev-server/src/graphql/resolvers/pokemon.js (hunk 6 above)
Comment (antoinechalifour) [reply dev-server/src/graphql/resolvers/pokemon.js:40]: The rest of the model uses `_` so we should use the same notation I guess

**Pull request merged**